recipes/funannotate/meta.yaml

```diff
@@ -17,64 +17,66 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.6,<3.9
-  run:
-    - biopython
-    - goatools
-    - matplotlib-base
-    - natsort
-    - numpy
-    - pandas
-    - psutil
     - python
-    - requests
-    - scikit-learn
-    - scipy
-    - seaborn
+  run:
+    - biopython  ==1.79
+    - goatools  ==1.1.6
+    - matplotlib-base  ==3.4.2
+    - natsort  ==7.1.1
+    - numpy  ==1.21.1
+    - pandas  ==1.3.1
+    - psutil  ==5.8.0
+    - python  >=3.6,<3.9
+    - requests  ==2.26.0
+    - scikit-learn  ==0.24.2
+    - scipy  ==1.5.3
+    - seaborn  ==0.11.1
     - blast ==2.2.31
-    - tantan
-    - bedtools
-    - hmmer
-    - exonerate
-    - diamond >=2.0.5
-    - tbl2asn
+    - tantan  ==26
+    - bedtools  ==2.30.0
+    - hmmer  ==3.3.2
+    - exonerate ==2.4.0
+    - diamond ==2.0.8
+    - tbl2asn  ==25.7
     - ucsc-pslcdnafilter
-    - pasa >=2.4.1
-    - trimmomatic
-    - raxml
-    - trimal
-    - mafft >=7
-    - iqtree
-    - kallisto >=0.46.0,<0.46.2
-    - minimap2
-    - evidencemodeler
-    - codingquarry
-    - stringtie
+    - pasa ==2.4.1
+    - trimmomatic  ==0.39
+    - raxml  ==8.2.12
+    - trimal  ==1.4.1
+    - mafft ==7.487
+    - iqtree  ==2.1.4_beta
+    - kallisto ==0.46.1
+    - minimap2  ==2.21
+    - evidencemodeler ==1.1.1
+    - codingquarry  ==2.0
+    - stringtie  ==2.1.7
     - snap
-    - glimmerhmm
-    - bamtools  # [linux]
-    - augustus  >3.3,<3.4  # [linux]
-    - trinity >=2.8.5=h8b12597_5  # [linux]
-    - trnascan-se
-    - hisat2
-    - proteinortho >=6.0.9
-    - salmon >=0.9
-    - perl
-    - perl-bioperl >1.7
-    - perl-dbd-mysql
-    - perl-clone
+    - glimmerhmm  ==3.0.4
+    - gmap  ==2017.11.15
+    - bamtools  ==2.5.1 # [linux]
+    - augustus  ==3.3.3  # [linux]
```

Comment:
From my local tests it feels like the slowness comes from this augustus version restriction, any reason not to update to 3.4.0?

Reply:
Yeah it's not compatible with the internal BUSCO method. What I can't figure out is that it seems to build if I specified the host python as 3.8, but I think that would limit it to work only if users were running 3.8? It does that not matter?

Reply:
I've made https://github.com/bioconda/bioconda-recipes/pull/30152 to try to rebuild the augustus 3.3.3, maybe that will help :crossed_fingers: